ansible/group_vars/all/secrets

```diff
@@ -1,11 +1,14 @@
 $ANSIBLE_VAULT;1.1;AES256
-39363263323231633231396432363163383464643230616461333630366234663232343835643933
```

Comment:
why has this changed?

Reply:
It adds vault_p3_monasca_agent_password to differentiate it from vault_monasca_agent_password.